src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -17,11 +17,13 @@ export class SavedObjectsClient {
       index,
       mappings,
       callCluster,
+      onBeforeWrite = () => {},
```

Comment:
Shouldn't this be a promise?

Reply:
Not necessary with `await`, promise/non-promise values all resolve the same.

Reply:
![giphy](https://user-images.githubusercontent.com/40265/31040841-14a82f94-a541-11e7-9056-00d454cef121.gif)